tests/test_multiannotator.py

```diff
@@ -0,0 +1,187 @@
+    s = s.apply(lambda x: x.mask(np.random.random(n) < 0.8))
+    s.dropna(axis=1, how="all", inplace=True)
+
+    # Estimate pred_probs
+    latent = count.estimate_py_noise_matrices_and_cv_pred_proba(
+        X=X_train,
+        labels=true_labels_train,
+        cv_n_folds=3,
+    )
+
+    row_NA_check = pd.notna(s).any(axis=1)
+
+    return {
+        "X_train": X_train[row_NA_check],
+        "true_labels_train": true_labels_train[row_NA_check],
+        "X_test": X_test[row_NA_check],
+        "true_labels_test": true_labels_test[row_NA_check],
+        "labels": s[row_NA_check].reset_index(drop=True),
+        "complete_labels": complete_labels,
+        "pred_probs": latent[4][row_NA_check],
+        "noise_matrix": noise_matrix,
+    }
+
+
+def make_data_long(data):
+    data_long = data.stack().reset_index()
+    data_long.columns = ["task", "annotator", "label"]
+
+    return data_long
+
+
+# Global to be used by all test methods. Only compute this once for speed.
+data = make_data()
+
+
+def test_convert_long_to_wide():
+    labels_long = make_data_long(data["labels"])
+    labels_wide = convert_long_to_wide_dataset(labels_long)
+
+    assert isinstance(labels_wide, pd.DataFrame)
+    # TODO: might add a test to see if the wide dataframe actually contains the right info
+
+
+# TODO: update this test to work with NaN values - randomly deleting data will make assertion invalid
+def test_get_worst_class():
+    labels = data["complete_labels"][0]  # only testing on first column
+    pred_probs = data["pred_probs"]
+
+    # Assert that the worst class index should be the class with the highest noise
+    assert dataset._get_worst_class(labels, pred_probs) == data["noise_matrix"].diagonal().argmax()
+
+
+def test_label_quality_scores_multiannotator():
+    labels = data["labels"]
+    pred_probs = data["pred_probs"]
+
+    lqs_multiannotator = get_label_quality_multiannotator(labels, pred_probs)
+    assert isinstance(lqs_multiannotator, pd.DataFrame)
+
+    # test verbose=False
+    lqs_multiannotator = get_label_quality_multiannotator(labels, pred_probs, verbose=False)
+    assert isinstance(lqs_multiannotator, pd.DataFrame)
+
+    # test passing a list into consensus_method
+    # TODO: change list items after adding more options
+    lqs_multiannotator = get_label_quality_multiannotator(
+        labels, pred_probs, consensus_method=["majority", "majority"]
+    )
+
+    # test different quality_methods
+    # also testing passing labels as np.ndarray
+    lqs_multiannotator = get_label_quality_multiannotator(
+        np.array(labels), pred_probs, quality_method="agreement"
+    )
+
+    # test returning annotator stats
+    lqs_annotatorstats = get_label_quality_multiannotator(
+        labels, pred_probs, return_annotator_stats=True
+    )
+    assert isinstance(lqs_annotatorstats, tuple)
+    assert isinstance(lqs_annotatorstats[0], pd.DataFrame)
+    assert isinstance(lqs_annotatorstats[1], pd.DataFrame)
```

Comment:
maybe check shape matches what we expect as well? And (some of the) column names?

Reply:
maybe also check every annotator is present in: `lqs_annotatorstats[1]`

Reply:
For both DFs, how about checking the values in each column (or row) lie within some bounds that we know they should satisfy, eg. nonnegative, between 0-1, etc.